_posts/2015-03-16-aerogear-ios-2.2.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title: AeroGear iOS SDK v2.2 is out !
```

Comment:
Okay, Andres already sent and update in https://github.com/aerogear/aerogear.org/pull/565